src/apps/profiles/admin.py

```diff
@@ -1,5 +1,12 @@
 from django.contrib import admin
+
 from .models import User
 
 
-admin.site.register(User)
+class UserAdmin(admin.ModelAdmin):
+    # The following two lines are needed:
```

Comment:
needed for what? what does this stuff do? the suspense is killing me!

Reply:
Psh. Outdated.

Reply:
![image](https://user-images.githubusercontent.com/21694127/97929309-b998fb80-1d1d-11eb-9f8c-a649737c355c.png)